tdrs-backend/docs/api/groupless_users.md

```diff
@@ -0,0 +1,75 @@
+# Groupless users list
+
+Get a list of users who have yet to be assigned to a role
+
+**Request**:
+
+`GET /v1/users/no_roles`
+
+Parameters:
+
+
+ A valid httpOnly cookie in the request header to track the users session
+
+*Note:*
+
+- Authorization Protected 
+
+**Response**:
+
+```json
+Content-Type: Application/json
+200 OK
+
+[
+  {
+    "id":"1fe8525c-5e91-4adb-8eb6-a0d3152a3fff",
+    "username":"user@example.com",
+    "first_name":"User",
+    "last_name":"Name"
+  },
+  {
+    "id":"a6b3d2ff-e5e6-3f9a-8eb6-b0a53fffafff",
+    "username":"user@example.com",
+    "first_name":"Jane",
+    "last_name":"Doe"
+  },
+  {
+    "id":"b0a5525c-e6e5-ff9a-8eb6-3fffaf3fb0a5a6b3",
+    "username":"user@example.com",
+    "first_name":"Jane",
+    "last_name":"Doe"
+  }
+]
```

Comment:
I think this list should include the STT as the OFA Analysts and Admins will need to know that to help identify the user for approval and role assignment